@LinProp/LinProp.m

```diff
@@ -1,42 +1,55 @@
-% Metas.UncLib.Matlab.LinProp V2.5.4
-% Michael Wollensack METAS - 10.05.2022
-% Dion Timmermann PTB - 18.05.2022
-%
-% LinProp Const:
-% a = LinProp(value)
-%
-% LinProp Input (RealUncNumber)
-% a = LinProp(value, standard_unc, (idof))
-%
-% LinProp Input (RealUncNumber)
-% a = LinProp(value, standard_unc, description)
-%
-% LinProp Input (ComplexUncNumber)
-% a = LinProp(value, [covariance], (description))
-%
-% LinProp Input (RealUncArray)
-% [a] = LinProp([value], [covariance], (description))
-%
-% LinProp Input (ComplexUncArray)
-% [a] = LinProp([value], [covariance], (description))
-%
-% LinProp From Samples
-% a = LinProp([samples], 'samples', (description), (probability))
+% This class supports the creation of uncertainty objects and subsequent
+% calculation with them as well as storage of the results. It can handle
+% complex-valued and multivariate quantities. Internally, LinProp objects
+% in MATLAB are wrappers of the .NET interface of the <a href="www.metas.ch/unclib">METAS UncLib library</a>.
 %
```

Comment:
Should we add as well a link to the GiHub Repro?
https://github.com/wollmich/metas-unclib-matlab-wrapper

Reply:
I do not see the benefit for "normal users". The repo provides access to the newest code versions - but these sometimes depend on the uncLib binaries, so they do not always work. 
Also, I think the documentation is getting too long, making it difficult to find the relevant info.

I think metioning the GitHub Repo somewhere on the uncLib homepage would be better.

(But I have not strong feelings about all of this.)